BREAKING.md

```diff
@@ -112,6 +113,9 @@ This process is supposed to be a catch-call case for cases where listeners did n
 If possible, it's recommended for DDS event listeners to not throw exceptions, but rather handle them appropriately without involving DDS itself.
 The purpose of this change to ensure that data model stays always synchronized with data projection that event listeners are building. If event listener is not able to fully / correctly process change event, that likely means data synchronization is broken and it's not safe to continue (and potentially, corrupt document).
 
+### Replace `innerRequestHandler` with `rootDataStoreRequestHandler`
+`innerRequestHandler` is removed from `@fluidframework/request-handlers` package and replaced with `rootDataStoreRequestHandler`.If a container wants to permit requests for specific data stores, they can add `rootDataStoreRequestHandler` to the list of request handlers they pass to the runtime factory. This will restore the ability to request root data stores. This will only work for root data stores, and if support is required for requesting non-root data stores and/or for legacy document support, a request handler can be created of their own that calls `runtime.IFluidHandleContext.resolveHandle(request)` just like `innerRequestHandler` used to do.
```

Comment:
Similar to above, just want to stress that the default action should be to do nothing unless the capabilities are strictly needed for the scenario.

```suggestion
`innerRequestHandler` is removed from `@fluidframework/request-handlers` package, and its usage is removed from `BaseContainerRuntimeFactory` and `ContainerRuntimeFactoryWithDefaultDataStore`.  If you are using these container runtime factories, attempting to access internal data stores via `request()` will result in 404 responses.

If you rely on `request()` access to internal root data stores, you can add `rootDataStoreRequestHandler` to your list of request handlers on the runtime factory.

It is not recommended to provide `request() access to non-root data stores, but if you currently rely on this functionality you can add a custom request handler that calls `runtime.IFluidHandleContext.resolveHandle(request)` just like `innerRequestHandler` used to do.
```